src/components/pages/room.yml

```diff
@@ -11,3 +11,6 @@ room:
   cancelRoomMembership:
     en: Cancel room membership
     de: Raum-Mitgliedschaft beenden
+  documentInputs:
+    en: Document inputs
+    de: Dokumenteingabe
```

Comment:
Maybe let's also put the plural here in German "Dokumenteingaben"?

Reply:
Done, thanks!